joomla/joomla-cms
# [4.0] Fix background shorthand incompatibilities

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/27572
Replaces https://github.com/joomla/joomla-cms/pull/27798

### Summary of Changes
Separating background color and background image in CSS to allow loading the specific .svg in Safari and Edge.


### Testing Instructions
Use Safari or Windows Edge.
Log in administration area.

Go to Articles.
Click Filter Options.
Click Select Condition dropdown and select a condition.
Selected item has no down arrow.
Apply PR. 
USE NPM
Repeat steps above.
Selected items have down arrow.
Confirm with other browsers such as Firefox, Chrome work correctly as before.

Repeat for RTL.

### Before patch
![27572-before](https://user-images.githubusercontent.com/368084/73796472-20bf5a80-4762-11ea-8474-fa970c47dc17.jpg)

### After patch
![27572-pr](https://user-images.githubusercontent.com/368084/73796489-2fa60d00-4762-11ea-9298-1564e86c5421.jpg)


UPDATED as it should now work in Safari
